src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, setTab }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            const changeTab = () => {
+              if (selectedTab !== id) {
```

Comment:
Comparing selectedTab (which is an object) with id (which is a property of object) will always result false, so setTab will always be called. Instead compare 'selectedTab.id' with 'id'.